influxdb2_client/src/models/label.rs

```diff
@@ -2,6 +2,88 @@
 
 use serde::{Deserialize, Serialize};
 
+/// Post create label request, to create a new label
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LabelCreateRequest {
+    /// Organisation ID
+    #[serde(rename = "orgID")]
+    pub org_id: String,
+    /// Organisation name
+    pub name: String,
+    /// Key/Value pairs associated with properties.
```

Comment:
I think this is supposed to say:

```suggestion
    /// Key/Value pairs associated with this label.
```

Reply:
Yeah! I'll change that